scripts/build/.variables

```diff
@@ -71,7 +71,7 @@ if [ "$CGO_ENABLED" = "1" ] && [ "$(go env GOOS)" != "windows" ]; then
 fi
 export GO_BUILDMODE
 
-GO_LDFLAGS="${GO_LDFLAGS:-} -w"
+GO_LDFLAGS="${GO_LDFLAGS:-}"
```

Comment:
https://github.com/docker/cli/pull/3717#issuecomment-1194551026

> Ok so after bisect we actually [strip DWARF symbol table in 20.10](https://github.com/docker/cli/blob/9d28e08a2dc7e4decf0eea363f6910fd8bde58eb/scripts/build/.variables#L17). So not a regression but this is needed for [docker/compose-cli#2151](https://github.com/docker/compose-cli/pull/2151). Needs to noop `llvm-strip` for `darwin/amd64` platform with glibc as discussed with @tonistiigi. Can't repro with alpine and LLVM 12.


Did a quick search on "why" the `-w` was added by default; I see the `-w` was added in https://github.com/docker/cli/commit/a787cbc93bf6571ea7186e372bfffddf682f6f61. Before that commit, a `scripts/build/ldflags` was sourced, which didn't have the option set. The commit is part of https://github.com/docker/cli/pull/78, but that specific commit is described as only "clean up" (but actually changed options).

There's no mention _at all_ about the change, and I can only assume reviewers missed it as it was part of a "clean up" that removed `scripts/build/ldflags` in favor of inlining it.